test/bin/ci_phase_iso_boot.sh

```diff
@@ -9,6 +9,8 @@ SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # shellcheck source=test/bin/common.sh
 source "${SCRIPTDIR}/common.sh"
 
+ENABLE_REGISTRY_MIRROR=${ENABLE_REGISTRY_MIRROR:-true}
```

Comment:
Need to disable mirror forcefully on aarch64 platform as we do not have the quay package for it.